frontend/src/features/VigilanceArea/useCases/filters/isVigilanceAreaPartOfCreatedBy.ts

```diff
@@ -0,0 +1,12 @@
+import type { VigilanceArea } from '@features/VigilanceArea/types'
+
+export function isVigilanceAreaPartOfCreatedBy(
+  vigilanceArea: VigilanceArea.VigilanceArea,
+  createdBy: string[] | undefined
+): boolean {
+  if (!createdBy || createdBy.length === 0) {
+    return true
+  }
+
+  return !!createdBy.includes(vigilanceArea.createdBy ?? '')
```

Comment:
suggestion: includes renvoie deja un boolean
```suggestion
  return createdBy.includes(vigilanceArea.createdBy ?? '')
```